### ISSUE TITLE
About culling down ACES versions.

### ISSUE THREAD

**Issue by KelSolaar** [imageworks/OpenColorIO-Configs#13]
I would be keen on discussing about potentially removing the various ACES config versions in the ImageWorks repository and keep a single one that would maybe be a *git-submodule* hosted on an AMPAS dedicated repository.

@aforsythe, @scoopxyz, any thoughts?

**Comment by scoopxyz** [comment]
I'm all for getting just the current ACES config in the master repo.

I would like to maintain the older versions by keeping a separate branch called "legacy". This should substantially decrease the repo size and make it quite clear to users what they should grab.

**Comment by KelSolaar** [comment]
> This should substantially decrease the repo size and make it quite clear to users what they should grab.

I could be wrong but without re-writing history I'm not sure the gains will be significant as you will still have the files in a blob within git tree.

**Comment by lgritz** [comment]
Download of the zip will decrease, but the repo size (what you get if you `git clone`) will not. Even a local git repo has the full history of the project.

**Comment by lgritz** [comment]
I'm all for removing the old ACES configs, and for that matter, even putting the extant ones in git LFS rather than in the main repo.

**Comment by dekekincaid** [comment]
While you are at it there still isn’t even a tagged released for aces
1.0.0-1.0.3 on either the Sony or the AMPAS repo.  The last one was a
1.0_r2.  We still grab it from HP’s repo because he has all versions
properly tagged and released there.  It would be good to have these
properly setup.

**Comment by scoopxyz** [comment]
> Download of the zip will decrease, but the repo size (what you get if you git clone) will not
Yes, of course, but there are about 4-5 people doing development work on the repo, the majority use case is just downloading the files to use. I'll need to read a bit more about LFS to see how it integrates into things.

> there still isn’t even a tagged released for aces
Yes, there was a sidebar conversation that prompted this ticket. Just needs to be done.

**Comment by KelSolaar** [comment]
As highlighted [in this comment](https://github.com/imageworks/OpenColorIO/issues/537#issuecomment-389111744), it should be possible to trim some fast by reducing indentation of the spi1d LUTs, some have 8 or 9 characters which is simply overkill.

Cheers,

Thomas
